content/YubiHSM2/Usage_Guides/YubiHSM_2_Windows_Deployment_Guide/Configure_the_Primary YubiHSM_2_Device.adoc

```diff
@@ -0,0 +1,175 @@
+
+
+== Configure the Primary YubiHSM 2 Device
+
+The YubiHSM Setup program is used to perform the initial configuration of the primary YubiHSM 2 device. The YubiHSM 2 is configured so that the necessary key material is generated on the YubiHSM 2 device:
+
+* One wrap key
+* One application authentication key
+* One audit key
+
+The wrap key is split between a predetermined number of key custodians, and each share should be recorded by its custodian. The authentication key and the audit key are exported under wrap to a file located in the current working directory.
+
+
+=== Overview of Procedure
+
+The configuration steps to be performed after you have inserted the primary factory preset YubiHSM 2 device into the air-gapped system are set out briefly below. These steps are described in detail in the subsequent "Procedure: Configuring Primary YubiHSM 2."
+
+1. Set up communication between the YubiHSM 2 tools and the device.
+
+2. Start the configuration process.
+
+3. Authenticate to the YubiHSM 2 device.
+
+4. Specify whether you need to add RSA decryption.
+
+5. Enter the name of the domain in which you need the application authentication key and audit key to be available in.
+
+6. Create the wrap key. The wrap key is very important as it allows you to export and import objects from and to the device. For example, you would export and import objects for backup purposes, as described in https://Backup_Key_Material.html[Backup Key Material].
```

Comment:
the link for backuo key material should be "link:Backup_Key_Material.adoc" to link correctly (the file is also empty).

Reply:
Changed (thank you for providing the link to replace it with)